src/commands/mouseMove.ts

```diff
@@ -28,7 +28,7 @@ export async function realMouseMove(
 ) {
   const basePosition= getCypressElementCoordinates(
     subject,
-    "topLeft",
+    options.position,
```

Comment:
```suggestion
    options.position ?? "topLeft",
```

I think this is reasonable to have a default topLeft coordinate system and not sure somebody want's to actually use this option but I agree we need to fix it. Thanks.

Reply:
@dmtrKovalenko 
Thank you for pointing that out and the quick response!

I noticed in another part of the repo it seems like `center` is the default coordinate. Is it OK if this default is `topLeft`, or should they be consistent?

https://github.com/dmtrKovalenko/cypress-real-events/blob/e9ca39f3fe1b1e03d57728aa8b1140d98df65739/src/getCypressElementCoordinates.ts#L167-L173

Reply:
Yeah for most of actions we have center as base point, but it will be strange to force people to write negative coordinated (which is still possible) for mouse moves.

Reply:
@dmtrKovalenko Good point!